msal/src/main/java/com/microsoft/identity/client/statemachine/results/SignInResult.kt

```diff
@@ -0,0 +1,178 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+package com.microsoft.identity.client.statemachine.results
+
+import com.microsoft.identity.client.statemachine.BrowserRequiredError
+import com.microsoft.identity.client.statemachine.Error
+import com.microsoft.identity.client.statemachine.IncorrectCodeError
+import com.microsoft.identity.client.statemachine.PasswordIncorrectError
+import com.microsoft.identity.client.statemachine.UserNotFoundError
+import com.microsoft.identity.client.statemachine.states.AccountResult
+import com.microsoft.identity.client.statemachine.states.SignInCodeRequiredState
+import com.microsoft.identity.client.statemachine.states.SignInPasswordRequiredState
+
+/**
+ * Results for native sign in flows.
+ * TODO add documentation & links to learn.microsoft.com
```

Comment:
Let's address this TODO

Reply:
I have removed the TODO as it was added when there was no documentation in the class.